src/diffusers_interpret/attribution.py

```diff
@@ -9,15 +9,31 @@ def gradients_attribution(
     pred_logits: torch.Tensor,
     input_embeds: Tuple[torch.Tensor],
     attribution_algorithms: List[AttributionAlgorithm],
-    explanation_2d_bounding_box: Optional[Tuple[Tuple[int, int], Tuple[int, int]]] = None,
+    explanation_2d_bounding_box: Optional[Tuple[Tuple[int,
+                                                      int], Tuple[int, int]]] = None,
     retain_graph: bool = False
 ) -> List[torch.Tensor]:
+    """Compute gradients attribution.
+
+    Args:
+        pred_logits: The logits of the model's prediction.
+        input_embeds: The input embeddings.
+        attribution_algorithms: The attribution algorithms to use.
+        explanation_2d_bounding_box: The 2D bounding box to use for the explanation.
+        retain_graph: Whether to retain the computation graph.

```

Comment:
We also need to put the typing in the docstring, right?
```suggestion
        pred_logits (`torch.Tensor`): The logits of the model's prediction.
        input_embeds (`Tuple[torch.Tensor]`): A tuple with all the input embeddings.
        attribution_algorithms (`List[AttributionAlgorithm]`): A list of the attribution algorithms to use. One algorithm per input in `input_embeds`.
        explanation_2d_bounding_box (`Tuple[Tuple[int, int], Tuple[int, int]]`, optional): Tuple with the bounding box coordinates where the attributions were calculated for. The tuple is like (upper left corner, bottom right corner). Example: `((0, 0), (300, 300))`. Defaults to `None`, which means that the gradients are computed for the whole output image.
        retain_graph (bool): Whether to retain the computation graph. Defaults to `False`.

```